vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,166 @@
+     Check the `Effective Roles` column on right to ensure the correct roles have been assigned.
+
+![Entando ID Management Role Mapping](./images/custportal-identity-userrole.png)
+
+  5. Under `Groups`, assign roles to groups as needed. Roles are additive.
+
+## Managing the Customer Portal
+
+As administrator for the Customer Portal, you can create and manage: users, customers, projects, and subscriptions. You can also assign projects to users who have activated their account on Jira and have been assigned roles in the Entando Identity Management System. 
+
+The administrator needs a Jira account and the correct role assignment in the Entando ID Management System as well.
+
+![Customer Portal Landing Page](./images/custportal-landing-page.png)
+
+* **Create a Customer or Partner**\
+Creating a `Customer` or `Partner` is a similar process. Below are the steps for adding a `Customer`. Please follow the same procedure for adding a `Partner`.
+1. Click `Add a Customer`
+2. Fill in the details. Note the following:
+     * The `Customer Number` must be unique 
+     * The `Notes` field is visible only to Support and Admin users 
+
+ ![Customer Portal Add Customer](./images/custportal-add-customer.png)   
+
+* **Create & Assign Projects**
+1. From the landing page, click on any customer to see the associated Project list.
+2. To create a new Project, click `Add a Project`
+    * Enter the `Project` information
+    * Provide the Organization ID from Jira. Each project must have a unique Organization ID.
+    * Click `Save`
+3. To assign Projects
+	* Use the drop-down menu under `Action` on the right → `Manage Users`
+    * Select the user for the Project and click `Submit` 
+    
+* **Manage Partners and Subscriptions** \
+Use the drop-down menu under `Action` to manage Partners, and request and manage subscriptions. 
+
+## Using the Customer Portal
+
+All Customer Portal users need Jira Service Management credentials and role access provided by your organization administrator. You can request and track subscriptions as well as create and track service tickets.
+
+Once you login to the Customer Portal, you will see a list of customers. Click on any customer to see their details and track their projects.
+
+1. To create a `Service Ticket` for a project, use the `Action` pull-down menu  → `Open Ticket`
+
+![Customer Portal Open Ticket](./images/custportal-open-ticket.png)
+
+2. To request a Subscription or track Tickets, use the `Action` pull-down menu and select the corresponding option.
+
+## Application Details
+The Entando Customer Portal demonstrates a number of major features in the Entando platform including:
+
+* Keycloak integration for role based access controls
+* Integration with JIRA Service Management for a customizable ticketing system 
+* Micro frontends using React and Angular and co-existing on the same dashboard page
```

Comment:
@nshaw can you check these? I copied these over from the Standard Banking Demo and added the ones I knew of.

Reply:
@jyunmitch One of the points of the Standard Demo is to demonstrate the features of the Entando Platform. We don't have the same goal with other templates so we can adjust the language. You already have a good bullet list in the Overview - Key Features so I'm not sure this is needed.